bundle/manifests/argoproj.io_argocds.yaml

```diff
@@ -260,14 +260,15 @@ spec:
                               items:
                                 type: string
                               type: array
+                              x-kubernetes-list-type: atomic
                             secretName:
                               description: SecretName is the name of the secret used
-                                to terminate SSL traffic on 443. Field is left optional
```

Comment:
I wonder where this change is coming from?

Reply:
These mostly likely came from when I ran ```make manifests``` from a different version of operator-sdk than what was used for the code in master 

I wasn't sure about what it's impact would be so I left it in to be reviewed but I can undo these

Reply:
> when I ran make manifests using a different version of operator-sdk

I think this is generally a bad idea. The Operator SDK version used for development must match the one used in the corresponding branch.

Reply:
Yeah, understood 
I think John mentioned he's going to generate manifests again with v1. 11.0 so I can do the same now, along with the other changes

Reply:
I've reverted the changes to the argocd CRD, but for some reason it isn't reflecting in the changed files view of the PR

Reply:
Do we want to do it in this PR? I thought John was working on generating manifests with v1.11.0 in a different PR

Reply:
The update to v1.11.0 has already been merged: https://github.com/argoproj-labs/argocd-operator/pull/429.

Reply:
@iam-veeramalla @jaideepr97 are you on mac or linux?  I'm on mac. I don't see these changes when running `make manifests` on Jaideep's branch.  If Abhishek is seeing these changes when running `make manifests` on this branch, then something weird is going on and I'm wondering if it's related to the OS.

`make manifests` calls `bin/controller-gen`, which is downloaded by the Makefile, and should be at version 0.6.1 regardless of which operator-sdk version is being used.

Reply:
I'm confused, when I use 1.11.0 locally I also see these changes being made
however, when I look at what's in master I don't see these changes  and they aren't showing up in the diff here either

Reply:
I'm on linux @jopit

Reply:
I feel like as the Gitops team, we should just look at git as the source of truth here :sweat_smile: 
if the git diff with master looks okay then I guess we  can be sure this won't be introducing anything unwanted into master regardless of what's happening on our local machines?

Reply:
OK, I must have got confused somehow.  When I run `make manifests` on Jaideep's GITOPS-1304-upgrade-ingress branch, I do indeed see the changes getting made.  When I run it on master, I don't.

**However** it seems this change is due to changing `k8s.io/api/extensions/v1beta1` to `k8s.io/api/networking/v1` in `argocd_types.go`.  If I make the same change to `argocd_types.go` in my master branch, I see the change in the CRD.

So this change **should* be committed.

Reply:
I think the important part of the change is this
```
                                   type: array
+                                  x-kubernetes-list-type: atomic
                                 secretName:
```

Reply:
@jopit Do you know if these changes should go in `config > crd > bases> argoproj.io_argocds.yaml` or `bundle > manifests > argoproj.io_argocds.yaml` or both? Because earlier I was seeing them in the bundle, but now I'm only seeing them under config

Reply:
`make bundle` needs to be rerun to regenerate the files under the bundle folder

Reply:
@jaideepr97 We do need to understand how the change was introduced.